### PR TITLE
Introduce CIS RHEL9 profiles

### PR DESCRIPTION
**Pull request by marcusburghardt**
#### Description:

CIS released the `CIS Red Hat Enterprise Linux 9 Benchmark v1.0.0` in 2022-11-28.
As outcome of the Benchmark review, the CIS RHEL9 controlfile was created, the proper CIS reference was included in all relevant rules and the final CIS RHEL9 profiles were updated from `Draft` to the final version.

Some rules, besides the CIS RHEL9 reference, might have RHEL CCEs included and the `prodtype` incremented.

#### Rationale:

Final version of CIS RHEL9 profiles.

#### Review Hints:

The PR is mostly about references inclusions. However, it was split in some commits aligned to the CIS Benchmark sections.

### CODE REVIEW

**Comment by Mab879** [comment]
I have not finished my review, but can you please add a CODEOWNERS entry for this new control file?

**Comment by marcusburghardt** [comment]
> I have not finished my review, but can you please add a CODEOWNERS entry for this new control file?

Sure. Done in the last commit.

**Comment by Mab879** [comment]
Please take a look at the output from `utils/refchecker.py`

```
$ ./utils/refchecker.py rhel9 cis_workstation_l2 cis 
Rule accounts_maximum_age_login_defs lacks required reference cis or cis@rhel9
Rule accounts_password_pam_retry lacks required reference cis or cis@rhel9
Rule accounts_passwords_pam_faillock_unlock_time lacks required reference cis or cis@rhel9
Rule configure_crypto_policy lacks required reference cis or cis@rhel9
Rule dconf_db_up_to_date lacks required reference cis or cis@rhel9
Rule enable_authselect lacks required reference cis or cis@rhel9
Rule set_password_hashing_algorithm_systemauth lacks required reference cis or cis@rhel9
```

**Review by Mab879 — APPROVED** [review]
Thanks for the PR! This going to great addition to the project.

**Review by Mab879 — CHANGES_REQUESTED** [review]
I am working on getting PR out for a script that checks for control id vs. rule reference discrepancies. But for the time being, I will place the output of it here:

```
sudo_require_reauthentication cis@rhel9 5.3.6 does not match the control id 5.3.5
```

File: linux_os/guide/system/software/sudo/sudo_require_reauthentication/rule.yml [hunk 1]
```diff
@@ -33,6 +33,7 @@ identifiers:
 
 references:
     cis@rhel8: 5.3.6
+    cis@rhel9: 5.3.6
```
Comment (Mab879) [inline linux_os/guide/system/software/sudo/sudo_require_reauthentication/rule.yml:36]:
```suggestion
    cis@rhel9: 5.3.5,5.3.6
```

Based on the control file it should be this.

File: linux_os/guide/system/accounts/accounts-pam/set_password_hashing_algorithm/set_password_hashing_algorithm_systemauth/rule.yml [hunk 2]
```diff
@@ -57,6 +57,7 @@ references:
     cis@alinux3: 5.5.4
     cis@rhel7: 5.4.3
     cis@rhel8: 5.4.4
+    cis@rhel9: 5.4.4
```
Comment (Mab879) [inline linux_os/guide/system/accounts/accounts-pam/set_password_hashing_algorithm/set_password_hashing_algorithm_systemauth/rule.yml:60]:
```suggestion
    cis@rhel9: 5.5.4
```

File: linux_os/guide/system/software/sudo/sudo_require_reauthentication/rule.yml (hunk 1 above)
Comment (marcusburghardt) [reply linux_os/guide/system/software/sudo/sudo_require_reauthentication/rule.yml:36]: Correct. Thanks

File: linux_os/guide/system/accounts/accounts-pam/set_password_hashing_algorithm/set_password_hashing_algorithm_systemauth/rule.yml (hunk 2 above)
Comment (marcusburghardt) [reply linux_os/guide/system/accounts/accounts-pam/set_password_hashing_algorithm/set_password_hashing_algorithm_systemauth/rule.yml:60]: Correct. Thanks

**Comment by marcusburghardt** [comment]
Thanks for the detailed review @Mab879 . I believe is everything adjusted now.

**Review by Mab879 — APPROVED** [review]
Thanks for fixing those last issues. LGTM.

**Pull request merged**